docs/khds/common/file/type/sseq.md

```diff
@@ -0,0 +1,93 @@
+| 0x93 | 4 bytes. 1st byte is track number [0..15]. The other 3 bytes are the relative adress of track data. Add nDataOffset (usually 0x1C) to find out the absolute address. | SSEQ is similar to MIDI in that track data are stored one after one track. Unlike mod music.
+| 0x00 .. 0x7f | Velocity: 1 byte [0..127]. Duration: Variable Length | NOTE-ON. Duration is expressed in tick. 48 for quartet note. Usually it is NOT a multiple of 3.
+| 0x80 | Duration: Variable Length | REST. It tells the SSEQ-sequencer to wait for a certain tick. Usually it is a multiple of 3.
+| 0x81 | Bank & Program Number: Variable Length | bits[0..7] is the program number, bits[8..14] is the bank number. Bank change is seldomly found, so usually bank 0 is used.
+| 0x94 | Destination Address: 3 bytes (Add nDataOffset (usually 0x1C) to find out the absolute address.) | JUMP. A jump must be backward. So that the song will loop forever.
+| 0x95 | Call Address: 3 bytes (Add nDataOffset (usually 0x1C) to find out the absolute address.) | CALL. It's like a function call. The SSEQ-sequncer jumps to the address and starts playing at there, until it sees a RETURN event.
+| 0xFD | NONE	| RETURN. The SSEQ will return to the caller's address + 4 (a Call event is 4 bytes in size).
+| 0xA0 .. 0xBf | See loveemu's sseq2mid for more details.	| Some arithmetic operations / comparions. Affect how SSEQ is to be played.
+| 0xC0 | Pan Value: 1 byte [0..127], middle is 64	| PAN
+| 0xC1 | Volume Value: 1 byte [0..127] | VOLUME
+| 0xC2 | Master Volume Value: 1 byte [0..127]	| MASTER VOLUME
+| 0xC3 | Value: 1 byte [0..64] (Add 64 to make it a MIDI value) | TRANSPOSE (Channel Coarse Tuning)
+| 0xC4 | Value: 1 byte | PITCH BEND
+| 0xC5 | Value: 1 byte | PITCH BEND RANGE
+| 0xC6 | Value: 1 byte | TRACK PRIORITY
+| 0xC7 | Value: 1 byte [0: Poly, 1: Mono]	| MONO/POLY
+| 0xC8 | Value: 1 byte [0: Off, 1: On] | TIE (unknown)
+| 0xC9 | Value: 1 byte | PORTAMENTO CONTROL
+| 0xCA | Value: 1 byte [0: Off, 1: On] | MODULATION DEPTH
+| 0xCB | Value: 1 byte | MODULATION SPEED
+| 0xCC | Value: 1 byte [0: Pitch, 1: Volume, 2: Pan] | MODULATION TYPE
+| 0xCD | Value: 1 byte | MODULATION RANGE
+| 0xCE | Value: 1 byte | PORTAMENTO ON/OFF
+| 0xCF | Time: 1 byte	| PORTAMENTO TIME
+| 0xD0 | Value: 1 byte | ATTACK RATE
+| 0xD1 | Value: 1 byte | DECAY RATE
+| 0xD2 | Value: 1 byte | SUSTAIN RATE
+| 0xD3 | Value: 1 byte | RELEASE RATE
+| 0xD4 | Count: 1 byte (how many times to be looped) | LOOP START MARKER
+| 0xFC | NONE	| LOOP END MARKER
+| 0xD5 | Value: 1 byte | EXPRESSION
+| 0xD6 | Value: 1 byte | PRINT VARIABLE (unknown)
+| 0xE0 | Value: 2 byte | MODULATION DELAY
+| 0xE1 | BPM: 2 byte | TEMPO
+| 0xE3 | Value: 2 byte | SWEEP PITCH
+| 0xFF | NONE | EOT: End Of Track
+
+
+
+
+
+
+
```

Comment:
1 new line character behind line 86 is enough ;)

Reply:
Whoops. I forgot to delete that cause I was rushing this